### DISCUSSION TITLE
Remove osrmti-en.json from transifex

### DISCUSSION THREAD

**Discussion by eis-ioki** [valhalla/valhalla#4850] [General]
I noticed, that there is a file called `osrmti-en.json` on the transifex translation website. It is not being used anywhere as far as I can tell. Not translating that file gives the impression, that a particular language is not completely translated even though `en-US.json` is at a 100%. Would it be possible to remove that file? It appears to be a verbatim copy of this file: https://github.com/Project-OSRM/osrm-text-instructions/blob/master/languages/translations/en.json

**Comment by kevinkreiser** [comment]
yeah we probably could. We will put it on the list of cleanup items
